### PR TITLE
Release 1.0.0-alpha.0 (cennznet/cennznet:1.0.0-rc1)

### PR DESCRIPTION
**Pull request by benxgao**
Version bumped by running:
```bash
 yarn run lerna version 1.0.0-alpha.0 --preid alpha --yes --no-git-tag-version --no-push
```

Tag added by running:
```bash
git tag -a 1.0.0-alpha.0
git push --tags
```

### CODE REVIEW

File: packages/crml-cennzx-spot/package.json [hunk 1]
```diff
@@ -19,12 +19,12 @@
   },
   "dependencies": {
     "@cennznet/api": "^0.20.7",
-    "@cennznet/crml-generic-asset": "^0.20.7",
+    "@cennznet/crml-generic-asset": "^1.0.0-alpha.0",
     "@cennznet/types": "^0.20.7",
```
Comment (bengooo) [inline packages/crml-cennzx-spot/package.json]: @AlexZhenWang I suppose the script bump version for `@cennznet/types`, do you understand why it did not?

Comment (holygits) [reply packages/crml-cennzx-spot/package.json]: Yeah, needs fixing

Comment (AlexZhenWang) [reply packages/crml-cennzx-spot/package.json]: But it works for other packages? What script did you use?

File: packages/api/package.json [hunk 2]
```diff
@@ -19,9 +19,9 @@
     "doc": "npx compodoc --tsconfig ../../tsconfig.json --hideGenerator --theme readthedocs --includes compodoc --output ../../docs/api"
   },
   "dependencies": {
-    "@cennznet/crml-attestation": "^0.20.7",
-    "@cennznet/crml-cennzx-spot": "^0.20.7",
-    "@cennznet/crml-generic-asset": "^0.20.7",
+    "@cennznet/crml-attestation": "^1.0.0-alpha.0",
+    "@cennznet/crml-cennzx-spot": "^1.0.0-alpha.0",
+    "@cennznet/crml-generic-asset": "^1.0.0-alpha.0",
     "@cennznet/types": "^0.20.7",
```
Comment (AlexZhenWang) [inline packages/api/package.json]: sorry if I am wrong, should this be `^1.0.0-alpha.0`?

File: packages/crml-attestation/package.json [hunk 3]
```diff
@@ -15,10 +15,10 @@
     "dependencies": {
         "@cennznet/api": "^0.20.7",
```
Comment (AlexZhenWang) [inline packages/crml-attestation/package.json]: Do we want these to be `^1.0.0-alpha.0`?

File: packages/crml-cennzx-spot/package.json [hunk 4]
```diff
@@ -19,12 +19,12 @@
   },
   "dependencies": {
     "@cennznet/api": "^0.20.7",
```
Comment (AlexZhenWang) [inline packages/crml-cennzx-spot/package.json]: same as above, maybe I am wrong

File: packages/crml-cennzx-spot/package.json (hunk 1 above)
Comment (AlexZhenWang) [inline packages/crml-cennzx-spot/package.json]: same

File: lerna.json [hunk 5]
```diff
@@ -1,8 +1,15 @@
 {
+  "command": {
```
Comment (AlexZhenWang) [inline lerna.json]: hi may I ask the reason for adding this?

File: packages/crml-cennzx-spot/package.json (hunk 1 above)
Comment (benxgao) [reply packages/crml-cennzx-spot/package.json]: It was suggested by using [plugnet-dev-version](https://github.com/plugblockchain/dev.js/blob/master/packages/dev/scripts/plugnet-dev-version.sh); As we are removing plugnet, I ran the script directly like ` yarn run lerna version 1.0.0-alpha.0 --preid alpha --yes --no-git-tag-version --no-push`

File: lerna.json [hunk 6]
```diff
@@ -1,8 +1,15 @@
...
+    "version": {
+      "allowBranch": [
+        "release/*"
```
Comment (benxgao) [inline lerna.json]: allow running lerna scripts in release/* branches

File: packages/crml-cennzx-spot/package.json (hunk 1 above)
Comment (AlexZhenWang) [reply packages/crml-cennzx-spot/package.json]: ah, thanks Ben, let me have a look

File: lerna.json (hunk 6 above)
Comment (AlexZhenWang) [reply lerna.json]: ah got it. Thanks Ben.

File: packages/crml-attestation/package.json (hunk 3 above)
Comment (benxgao) [reply packages/crml-attestation/package.json]: Yes, I changed `0.20.7` to be `1.0.0-alpha.0` manually in a following commit, and I am trying to test it locally.

Comment (AlexZhenWang) [reply packages/crml-attestation/package.json]: Thanks Ben!

**Pull request closed without merging**